tests/components/post_deleted_modal.test.jsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+import {Modal} from 'react-bootstrap';
+
+import {mountWithIntl} from 'tests/helpers/intl-test-helper.jsx';
+import PostDeletedModal from 'components/post_deleted_modal.jsx';
+
+describe('components/ChannelInfoModal', () => {
+    test('should match snapshot when modal is showing', () => {
+        function emptyFunction() {} //eslint-disable-line no-empty-function
+
+        const wrapper = shallow(
+            <PostDeletedModal
+                show={true}
+                onHide={emptyFunction}
+            />
+        );
+
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('should match snapshot when modal is not showing', () => {
+        function emptyFunction() {} //eslint-disable-line no-empty-function
+
+        const wrapper = shallow(
+            <PostDeletedModal
+                show={false}
+                onHide={emptyFunction}
+            />
+        );
+
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('should call onHide callback when modal is hidden', (done) => {
+        function onHide() {
+            done();
+        }
```

Comment:
No change require for now but we are updating patterns and taking advantage of jest in spying/mocking function calls similar to:
```
const onHide = jest.fn();
...
expect(onHide).toHaveBeenCalledTimes(1); // or expect(onHide).toHaveBeenCalled();
```